src/filters/mentions.cjs

```diff
@@ -1,92 +1,72 @@
+  const url = new URL(input);
+  url.search = url.hash = '';
+  const output = url.toString();
+  if (!output.endsWith('/')) {
+    return `${output}/`;
+  }
+  return output;
 };
 
-const likes = (mentions) =>
-  mentions.filter((entry) => entry['wm-property'] === 'like-of');
-
-const reposts = (mentions) =>
-  mentions.filter((entry) => entry['wm-property'] === 'repost-of');
-
-const webMentions = (mentions) => {
-  // define which types of webmentions should be included per URL.
-  // possible values listed here:
-  // https://github.com/aaronpk/webmention.io#find-links-of-a-specific-type-to-a-specific-page
-  const allowedTypes = ['mention-of', 'in-reply-to', 'bookmark-of'];
+const forUrl = (mentions, url) =>
+  mentions.children
+    .filter(
+      (entry) =>
+        normalizeURL(entry['wm-target']) === normalizeURL(url) &&
+        !entry['wm-private'],
+    )
+    .sort(orderByDate);
 
-  // clean webmention content for output
-  const clean = (entry) => {
-    const { html, text } = entry.content;
+// clean webmention content for output
+const clean = (entry) => {
+  const { html, text } = entry.content;
 
-    if (html) {
-      // really long html mentions, usually newsletters or compilations
-      entry.content.value = sanitizeHTML(
-        html.length > 2000
-          ? `
-            <p>${html.substring(0, 250)}…</p>
-            <p>
-              Read more:
-              <a href="${entry['wm-source']}">${entry['wm-source']}</a>
-            </p>
-          `
-          : html,
-      );
-    } else {
-      entry.content.value = sanitizeHTML(text);
+  if (html) {
+    let truncated = truncate(html, 50, { byWords: true, ellipsis: '…' });
+    // Strip non-alphanumeric trailing chars (e.g. commas, periods):
+    if (
+      truncated.endsWith('…') &&
+      truncated.slice(-2, -1).match(/[^A-Z|a-z|0-9]/) !== null
+    ) {
+      truncated = `${truncated.slice(0, -2)}…`;
     }
+    entry.content.value = sanitizeHTML(truncated);
+  } else {
+    entry.content.value = sanitizeHTML(text);
+  }
 
-    return entry;
-  };
+  return entry;
+};
 
-  // only allow webmentions that have an author name and a timestamp
-  const checkRequiredFields = (entry) => {
-    const { author, published, content } = entry;
-    return (
-      Boolean(author) &&
-      Boolean(author.name) &&
-      Boolean(published) &&
-      Boolean(content)
-    );
-  };
+// only allow webmentions that have an author name and a timestamp
+const checkRequiredFields = (entry) => {
+  const { author, published, content } = entry;
+  return Boolean(author && author.name && published && content);
+};
 
+const getTypes = (mentions, allow) =>
   // run all of the above for each webmention that targets the current URL
-  return mentions
-    .filter((entry) => allowedTypes.includes(entry['wm-property']))
+  mentions
+    .filter((entry) => (allow || defaultTypes).includes(entry['wm-property']))
     .filter(checkRequiredFields)
     .sort(orderByDate)
```

Comment:
@mirisuzanne If we're already sorting these in `forUrl`, do we need to sort them again here?

Reply:
Adjusted in https://github.com/oddbird/oddleventy/commit/31d21fca24310af09f2b0c6fcff30de9b578204e